{"uuid": "3b750f26-0de6-4eb3-b5e2-5bb9954d3905", "children": ["641dd799-7e60-492d-b34e-1b6e8cb9801b"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 80, in setup_profiles\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/fixtures_4x.py\", line 179, in setup_profiles\n    lf_tools.create_non_meh_dut(ssid_data=ssid_data_list)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 1126, in create_non_meh_dut\n    self.Create_Dut()\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 708, in Create_Dut\n    self.CreateDut.cv_test.sync_cv()\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/cv_test_manager.py\", line 481, in sync_cv\n    print(self.run_cv_cmd(cmd))\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/cv_test_manager.py\", line 487, in run_cv_cmd\n    self.json_post(req_url, data, debug_=False, response_json_list_=response_json)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/lfcli_base.py\", line 256, in json_post\n    json_response = lf_r.json_post(show_error=debug_,\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/LFRequest.py\", line 176, in json_post\n    resp = urllib.request.urlopen(myrequest)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 222, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 525, in open\n    response = self._open(req, data)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 542, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n  File \"/usr/lib/python3.8/urllib/request.py\", line 502, in _call_chain\n    result = func(*args)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1383, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1358, in do_open\n    r = h.getresponse()\n  File \"/usr/lib/python3.8/http/client.py\", line 1348, in getresponse\n    response.begin()\n  File \"/usr/lib/python3.8/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/lib/python3.8/http/client.py\", line 277, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/lib/python3.8/socket.py\", line 669, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1698487301367, "stop": 1698487331231}], "start": 1698487301367, "stop": 1698487331425}